components/Checkin.js

```diff
@@ -58,6 +58,7 @@ class Checkin extends React.Component {
         checkedStatus: 'Hold to check in'
       }
     } else {
+      console.log("this.checkinable()", this.checkinable())
```

Comment:
We can make an issue to go in and remove console.logs and comments

Reply:
Done!